public/Grant-DbaAgPermission.ps1

```diff
@@ -23,7 +23,7 @@ function Grant-DbaAgPermission {
         Only modify specific availability groups.
 
     .PARAMETER Type
-        Specify type: Endpoint or AvailabilityGroup. Endpoint will modify the DatabaseMirror endpoint type.
+        Specifies the type of object to which permissions will be granted. Acceptable values are "Endpoint" or "AvailabilityGroup". If "Endpoint" is specified, permissions will be granted to the Database Mirroring endpoint.
```

Comment:
```suggestion
        Specify the object type to grant the provided Permission. Values accepted are Endpoint and/or AvailabilityGroup. Endpoint type will update the DatabaseMirroring object type found on the instance.
```

The one word use of `DatabaseMirroring` is to indicate the object type name itself based on the code. Don't think that is confusing to folks or misleading.